.github/dependabot.yml

```diff
@@ -1,23 +1,8 @@
 version: 2
 updates:
-  # glean-core
+  # glean
   - package-ecosystem: "cargo"
-    directory: "/glean-core"
-    schedule:
-      interval: "daily"
-    ignore:
-      # Pinned to old version due to breaking changes
-      - dependency-name: "rkv"
-
-  # glean-ffi
-  - package-ecosystem: "cargo"
-    directory: "/glean-core/ffi"
-    schedule:
-      interval: "daily"
-
-  # glean benchmark
-  - package-ecosystem: "cargo"
-    directory: "/glean-core/benchmark"
+    directory: "/"
```

Comment:
Is it intentional that you do not ignore rkv anymore?

Reply:
Yes! We recently upgraded it (#1260)